### DISCUSSION TITLE
Can we integrate Flowise with our software

### DISCUSSION THREAD

**Discussion by lovepreetCodewalnut** [FlowiseAI/Flowise#662] [General]
Hello everyone,
I'm working on developing an application that allows users to upload multiple files, and then they can ask questions related to those files to get answers. The frontend UI will have a simple search box where users can enter their questions, and the answers will be displayed below.

For the backend, I plan to use Flowise, a tool that will handle the processing of the uploaded files and generate responses to user queries.

Additionally, I'd like to implement a button on the frontend to enable users to upload the files easily.

**Comment by HenryHengZJ** [comment]
here's a full tutorial on that - https://www.youtube.com/watch?v=kOwmPe8aLAA&t=1205s&ab_channel=MenloParkLab
